src/bootstrap/src/core/config/toml/rust.rs

```diff
@@ -389,9 +391,13 @@ pub fn check_incompatible_options_for_ci_rustc(
     Ok(())
 }
 
-pub(crate) const VALID_CODEGEN_BACKENDS: &[&str] = &["llvm", "cranelift", "gcc"];
+pub(crate) const KNOWN_CODEGEN_BACKENDS: &[&str] = &["llvm", "cranelift", "gcc"];
```

Comment:
Question: should this instead be called "built-in"? Not that I think that is really better.